src/crypto/base64.ts

```diff
@@ -14,8 +14,8 @@ class Base64 {
     return Utf8.stringify(CryptoBase64.parse(data));
   }
 
-  public static decodeJSON<T>(data: string): T {
-    return JSON.parse(Base64.decode(data));
+  public static decodeJSON<T>(data?: string): T {
+    return data ? JSON.parse(Base64.decode(data)) : '';
```

Comment:
I guess it's better to have the default value as an empty string
```suggestion
  public static decodeJSON<T>(data: string = ''): T {
    return JSON.parse(Base64.decode(data));
```

Reply:
the other possible solution is to do something like this:
```suggestion
  public static decodeJSON(data: undefined | null): null 
  public static decodeJSON<T>(data: string): T 
  public static decodeJSON<T>(data?: string | null): T | null {
    return data == null ? null : JSON.parse(Base64.decode(data));
```